### DISCUSSION TITLE
How to inline svg from `assets` folder?

### DISCUSSION THREAD

**Discussion by MrShnaider** [nuxt/nuxt#16717] [Questions]
Hello, everyone. I want to inline small svgs, about 500 bytes each, directly into css files so that they are not loaded individually. I insert these icons via `background: url()`, and store them in the `assets` folder. There were some plugins in the documentation for the Nuxt 2, but they are suitable for "webpack". What do I need to do to inline my icons in the Nuxt 3 and how to set the maximum size for this inline?

**Comment by palamarchukser** [comment]
https://stackoverflow.com/questions/72052807/how-to-use-any-icons-with-nuxt3/72055404#72055404

**Comment by mihail727** [comment]
or u can use https://www.npmjs.com/package/vite-svg-loader

**Comment by dpmango** [comment]
I have problems using vite-svg-loader.

Inline works great, but seems to be it breaks nuxt imports.

https://stackblitz.com/edit/github-mzuffh-5lgj4b?file=app.vue
<img width="412" alt="Снимок экрана 2022-09-30 в 00 32 48" src="https://user-images.githubusercontent.com/18535792/193146012-9ec2f375-05f5-4482-b1f9-1cd81534c5e6.png">
